site/docs/basics/controller.zh-CN.md

```
@@ -1106,3 +1112,10 @@ exports.security = {
 ```
 
 如果没有配置 `domainWhiteList` 或 `domainWhiteList` 数组为空，则默认允许所有跳转请求，等同于使用 `ctx.unsafeRedirect(url)`。
+=======
+| Controller 装饰器                                                | 说明                                                          |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)